lib/utils.pm

```diff
@@ -632,7 +632,7 @@ Examples:
 sub zypper_ar {
     my ($url, %args) = @_;
     my $name = $args{name} // '';
-    my $priority = $args{priority} // undef;
+    my $priority = $args{priority} // '';
```

Comment:
Why this? What it improves?


Also it changes meaning of:
```
my $prioarg = defined($priority) && !is_sle('<=12') ? "-p $priority" : "";
```
It'd have to be changed to
```
my $prioarg = $priority && !is_sle('<=12') ? "-p $priority" : "";
```
But I'd prefer it to stay, if it's only a style change.
Also it's an unrelated change, it should be changed in separate commit (in case it needs to be reverted).

Reply:
OK, I see it got broken in a1badaffb8c1c571015e9d055abfac39e4c22d4c. Could you please change it as
```
-    my $cmd_mr = "mr -p $priority $url";
+    my $cmd_mr = "mr $prioarg $url";
```
Put it into separate commit and add `Fixes: a1badaffb ("utils::zypper_ar(): Fix repo priority on SLE-11")` into commit message.

Reply:
*this change is ready for review*